pkg/model/model.go

```diff
@@ -5,13 +5,14 @@ import (
 )
 
 type Network struct {
-	ID                 uint                  `json:"id" gorm:"primarykey"`
-	CreatedAt          int64                 `json:"created_at" gorm:"autoCreateTime"`
-	UpdatedAt          int64                 `json:"updated_at" gorm:"autoUpdateTime"`
-	DeletedAt          soft_delete.DeletedAt `json:"-"`
-	ContractAddressURL string                `json:"contract_address_url"`
-	StateDumpURL       string                `json:"state_dump_url"`
-	Status             bool                  `json:"status"`
+	ID                  uint                  `json:"id" gorm:"primarykey"`
+	CreatedAt           int64                 `json:"created_at" gorm:"autoCreateTime"`
+	UpdatedAt           int64                 `json:"updated_at" gorm:"autoUpdateTime"`
+	DeletedAt           soft_delete.DeletedAt `json:"-"`
+	ContractAddressURL  string                `json:"contract_address_url"`
+	StateDumpURL        string                `json:"state_dump_url"`
+	LatestComponentType string                `json:"latest_component" gorm:"-"`
```

Comment:
Do we need this field? I think it can be useful later. But it can make some confusion now. Because our project team doesn't have perfect comprehension of process.

Reply:
Additionally, the dependency is not syncrous. `batch-submitter`, `explorer`, `relayer` can be created 
in any order after creating l2geth
1. deployer
2. data-transport-layer
3. l2geth
4. batch-submitter explorer relayer

Reply:
Thank you for you explanation. I updated!